### PR TITLE
Reg: TC test_voq_chassis_app_db_consistency.py Modification Due to Lag ID Set Changes

### PR DESCRIPTION
**Pull request by vikshaw-Nokia**
### Description of PR

Summary:
Fixes # (issue)
Due to LAG ID SET Changes https://github.com/sonic-net/sonic-buildimage/pull/20369.
The Test Case Expectation and Assertion need to be changed.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Changes by the Functionality https://github.com/sonic-net/sonic-buildimage/pull/20369 of Lag ID assignment.
The TC Expectation and assertion is changed

#### How did you do it?
Ignoring the SYSTEM_LAG_ID_SET to be same as in case pre-dump.
But rather be assigned from `SYSTEM_LAG_IDS_FREE_LIST` in order.
Added a sanity of lag_id_set to ensure the functionality of PR https://github.com/sonic-net/sonic-buildimage/pull/20369.

#### How did you verify/test it?
Tested and Verified on a T2 VOQ Chassis

### CODE REVIEW

File: tests/voq/test_voq_chassis_app_db_consistency.py [hunk 1]
```diff
@@ -252,11 +247,34 @@ def get_db_dump(duthosts, duthost):
...
+
+    def verify_system_lag_sanity():
+        seen = set(LAG_IDS_FREE_LIST + SYSTEM_LAG_ID_SET)
+        if len(seen) != (end - start + 1):
+            logging.error(
+                "Missing or extra values are found in SYSTEM_LAG_IDS_FREE_LIST:{} or SYSTEM_LAG_ID_SET:{}".
+                format(LAG_IDS_FREE_LIST, SYSTEM_LAG_ID_SET))
+            return False
+        if any(LAG_IDS_FREE_LIST.count(x) > 1 or SYSTEM_LAG_ID_SET.count(
+                x) > 1 or x in LAG_IDS_FREE_LIST and x in SYSTEM_LAG_ID_SET for x in seen):
```
Comment (arista-nwolfe) [inline tests/voq/test_voq_chassis_app_db_consistency.py:270]: Can we bracket the last two conditions to make it more obvious?
I.E.
```
if any(LAG_IDS_FREE_LIST.count(x) > 1 or SYSTEM_LAG_ID_SET.count(
           x) > 1 or (x in LAG_IDS_FREE_LIST and x in SYSTEM_LAG_ID_SET) for x in seen):
```

**Review by arista-nwolfe — COMMENTED** [review]
LGTM, just one comment

File: tests/voq/test_voq_chassis_app_db_consistency.py (hunk 1 above)
Comment (vikshaw-Nokia) [reply tests/voq/test_voq_chassis_app_db_consistency.py]: Hi @arista-nwolfe 
Have Made the Changes.
Please Review

**Review by arista-nwolfe — APPROVED** [review]
Thanks for making this change!

**Review by arlakshm — APPROVED** [review]

**Pull request merged**